.github/workflows/pull-request-action.yml

```diff
@@ -0,0 +1,42 @@
+name: Reusable PR testing for mac and linux
+
+on:
+  workflow_call:
+    inputs:
+      targets:
+        type: string
+        description: "Make and CTest targets. If not specified, everything is tested"
+        required: false
+
+jobs:
+  handle-syncwith:
+    name: Call Reusable SyncWith Handler
+    uses: NilFoundation/ci-cd/.github/workflows/reusable-handle-syncwith.yml@v1.2.0
```

Comment:
@v1.2.1 contains important fix, update to it

Reply:
Thanks!